__init__.py

```diff
@@ -164,7 +164,7 @@ def mappings_from(nm):
     vc = hass.data[VANTAGE_CONTROLLER]
 
     await hass.async_add_executor_job(
-        functools.partial(vc.load_xml_db, config.get(CONF_DISABLE_CACHE, False))
+        functools.partial(vc.load_xml_db, config.get(CONF_ENABLE_CACHE, False))
```

Comment:
Don't we need to !config.get(...) to invert the value now that we inverted the sense of the flag?  The arg to load_xml_db is disable_cache, still.

Reply:
You are right.  Serves me right for not testing this code properly.  ;-(

Fixed.  Also, I patched this into a new client and tested it this time.  :-)